### ISSUE TITLE
`train_results` accumulates results

### ISSUE THREAD

**Issue by qbilius** [neuroailab/tfutils#63]
I had `save_metrics_freq = 500`, so `train_results` were being saved every 500 steps, but each record turned out to contain 500 lines of `loss` and `learning_rate`, presumably corresponding to loss and lr at each step.

See dbname: tnn_mem, collname: trainable_mem_multiloss.files, exp_id: train_6.

**Comment by qbilius** [comment]
Apparently this is expected behavior.
